core/config/locales/en.yml

```diff
@@ -715,6 +716,18 @@ en:
           amount_authorized_exceeds_total_credit: " exceeds the available credit"
           amount_used_not_zero: "is greater than zero. Can not delete store credit"
           update_reason_required: "A reason for the change must be selected"
+      variants:
+        index:
+          show_deleted: Show deleted variants
+        new:
+          new_variant: New variant
+        edit:
+          edit_variant: "Edit variant - %{options_text}"
+        form:
+          dimensions: Dimensions
+          no_tax_category_override: No override
+          pricing: Pricing
+          pricing_hint: These values are populated from the product details page and can be overridden below
```

Comment:
This is what happens if you use the t('.something') convention. It's nice, and very well namespaced. We're losing a bit of generality here, but on the other hand it's very easy to find where to add a translation. Loved it.